### PR TITLE
Mobile 4329

### PR DESCRIPTION
**Pull request by crazyserver**

### CODE REVIEW

File: src/core/features/dataprivacy/services/dataprivacy.ts [hunk 1]
```diff
@@ -41,6 +42,10 @@ export class CoreDataPrivacyService {
             return false;
         }
 
+        if (site?.isFeatureDisabled(CORE_DATAPRIVACY_FEATURE_NAME)) {
+            return false;
+        }
```
Comment (dpalou) [inline src/core/features/dataprivacy/services/dataprivacy.ts:47]: This check isn't needed here. The links delegate automatically checks it using featureName, and the user delegate checks it automatically too using the handler name and the delegate name.

File: src/core/features/dataprivacy/tests/behat/create_data_request.feature [hunk 2]
```diff
@@ -59,4 +59,4 @@ Feature: Data export and delete from the privacy API
     When I run all adhoc tasks
     And I pull to refresh in the app
     And I should find "Download ready" near "Export all of my personal data" in the app
-    # TODO: Add download link and test it.
+    And I press "Download" in the app
```
Comment (dpalou) [inline src/core/features/dataprivacy/tests/behat/create_data_request.feature:62]: Shouldn't you test that the browser has been opened pointing to the site? You can use the exact URL or just that the site is opened, something like:

Then the app should have opened a browser tab with url "$WWWROOTPATTERN"

**Review by dpalou — CHANGES_REQUESTED** [review]

File: src/core/features/dataprivacy/services/dataprivacy.ts (hunk 1 above)
Comment (crazyserver) [reply src/core/features/dataprivacy/services/dataprivacy.ts]: Thanks, removed

File: src/core/features/dataprivacy/tests/behat/create_data_request.feature (hunk 2 above)
Comment (crazyserver) [reply src/core/features/dataprivacy/tests/behat/create_data_request.feature:62]: Done! waiting for tests

**Comment by crazyserver** [comment]
Hi Dani! I've added one more commit.

**Review by dpalou — APPROVED** [review]

**Pull request merged**